flocker/node/agents/blockdevice.py

```diff
@@ -1865,6 +2043,16 @@ class DuplicateFilesystemId(Exception):
     """
 
 
+def _blkid(*argv):
+    try:
+        return check_output([b"blkid"] + list(argv))
```

Comment:
:question: Capture stderr so that it's included in CalledProcessError and logged by Eliot rather than leaking and corrupting the logs.